storage/innobase/dict/dict0stats.cc

```diff
@@ -423,6 +423,8 @@ dict_stats_table_clone_create(
 	UT_LIST_INIT(t->freed_indexes, &dict_index_t::indexes);
 #endif /* BTR_CUR_HASH_ADAPT */
 
+	t->stats_error_printed = table->stats_error_printed;
```

Comment:
I think that you mentioned that you’re going to remove this function from 10.6, where `dict_stats_save()` should always be protected by a metadata lock and therefore the `table_orig` parameter can just be renamed to `table`.

Let’s push this one quickly to 10.5 and merge to 10.6, so that you can file a 10.6 pull request for this follow-up cleanup ASAP. That pull request should remove also the test that is added here, to avoid wasting CI resources.